lib/mwUtil.js

```diff
@@ -414,4 +414,86 @@ mwUtil.hydrateResponse = (response, fetch) => {
     .thenReturn(response);
 };
 
+/**
+ * Checks whether the date is today or in the past in UTC-0 timezone
+ *
+ * @param {Date} date a date to check
+ * @return {boolean} true if the date is in the past
+ */
+mwUtil.isHistoric = (date) => {
```

Comment:
Regardless of whether we will store historical data for the trending edits or not, :+1: for separating this from the feed context.